### PR TITLE
Update values file to provide a default image and tag for use during script-pod creation

### PR DESCRIPTION
**Pull request by rain-on**
Currently - a script pod will be either:
* kubernetes-agent-base-tools (by default), or
* an image as defined on the script execution command

When the kuberentes-agent is used as a worker, it is almost guaranteed that the default container will have insufficient tooling for most workflows - thus requiring the customer to define an execution-container on any/all steps which may be sent to a k8s worker.

This change introduces helm values which can be used to define the container used when spawning a script pod.

These values will be passed into the Tentacle containers as environment-variables - and will be dependent on whether the agent is operating as a worker or a deployment target.

### CODE REVIEW

File: charts/kubernetes-agent/values.yaml [hunk 1]
```diff
@@ -163,6 +163,18 @@ scriptPods:
     clusterRole:
       rules: []
 
+  deploymentTarget:
```
Comment (APErebus) [inline charts/kubernetes-agent/values.yaml:166]: These are missing comments. See the other image options for examples

Comment (rain-on) [reply charts/kubernetes-agent/values.yaml:168]: added comments

Comment (APErebus) [reply charts/kubernetes-agent/values.yaml:168]: These are at the wrong level, they should be above the `image` node (see the output)

File: .changeset/polite-donuts-jump.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"kubernetes-agent": minor
+---
+
+Allow the script-pod container to be defined in values
```
Comment (APErebus) [inline .changeset/polite-donuts-jump.md:5]:
```suggestion
Allow the container used by script pods to be modified for both worker and deployment target
```

**Review by APErebus — CHANGES_REQUESTED** [review]

File: charts/kubernetes-agent/values.yaml (hunk 1 above)
Comment (rain-on) [reply charts/kubernetes-agent/values.yaml:166]: resolved

File: charts/kubernetes-agent/values.yaml [hunk 3]
```diff
@@ -163,6 +163,22 @@ scriptPods:
...
+    # -- The repository, pullPolicy & tag to use for the script-pod image used when the cluster is a deploymentTarget
```
Comment (APErebus) [inline charts/kubernetes-agent/values.yaml:167]:
```suggestion
    # -- The repository, pullPolicy & tag to use for the script pod image when the agent is a deployment target
```

File: charts/kubernetes-agent/values.yaml [hunk 4]
```diff
@@ -163,6 +163,22 @@ scriptPods:
...
+    # @section -- Script pod values
+    image:
+      repository: ""
+      pullPolicy: ""
+      tag: ""
+      
+  worker:
+    # -- The repository, pullPolicy & tag to use for the script-pod image used when running as a worker
```
Comment (APErebus) [inline charts/kubernetes-agent/values.yaml:175]:
```suggestion
    # -- The repository, pullPolicy & tag to use for the script pod image when the agent is a worker
```

**Review by APErebus — COMMENTED** [review]
Couple of minor spelling issues and then we'll need to update the tentacle tag when the tentacle PR is merged

**Review by APErebus — APPROVED** [review]
Excellent work mate!

**Pull request merged**